### ISSUE TITLE
positioning multiple charges on an ordinary

### ISSUE THREAD

**Issue by wfraser** [drawshield/Drawshield-Code#101]
It seems to be impossible to position two different charges on an ordinary.

The closest I can get is using `between`, but that means one of them is duplicated:

`argent on a chief azure a harp between two stags heads or`

This positions the three charges on the ordinary as you'd expect.

If you change `between` to `beside`, though:

`argent on a chief azure a harp beside a stags head or`

The harp gets positioned on the chief, but the stags head does not.

Also, the `between` only works if written `on a <ordinary> <charge> between <charge>`. If you write `<charge> between <charge> on <ordinary>` it does not work, even though the simple `<charge> on <ordinary>` in that same order works fine. Basically the positioning of charges on ordinaries is super inconsistent in what works and what doesn't.

**Comment by karlwilcox** [comment]
Yes, this is true. The current approach requires custom code for each type of placement "on" an ordinary and I haven't written it for very many situations.

I would like to rewrite the whole placement code (the currently unused file placement-data.inc is a very early start) but haven't got a lot of time at the moment - it is also the case that I wouldn't want to switch to the new code until it is at least as capable as the existing algorithm.

However, I will see what I can in the short(ish) term...
